### DISCUSSION TITLE
Issue in displaying XML Requests and Response bodies

### DISCUSSION THREAD

**Discussion by malathikrishnap** [rohit-gohri/redocusaurus#358] [General]
Hi there, I'm noticing that Redocusaurus is not parsing the xml samples. It just displays "No Sample" although sample exists.
The same behaviour exists here as well: https://redocusaurus.vercel.app/docs/nested/nested-1

Is there anything that I can do to render the XML samples? or is it a limitation with Redocusaurus tool?
Also, I'd like to add cURL and Python as the other content types. Please let me know if there are any known issues around these.

Btw, I found an old issue on the same subject: #170 

Any help will be highly appreciable. Thank you!

**Comment by rohit-gohri** [comment]
Does it work if you try it in https://github.com/Redocly/redoc/ ?

**Reply by malathikrishnap** [reply]
I need to try. But, when I try my spec in the swagger editor, all the samples are displayed just fine.
XML and responses as well.

In Redocusarus, the responses were also not getting displayed.

**Reply by malathikrishnap** [reply]
No, it doesn't work in  https://github.com/Redocly/redoc/ either.

**Reply by rohit-gohri** [reply]
> No, it doesn't work in [Redocly/redoc](https://github.com/Redocly/redoc/) either.

Then unfortunately can't do much, as Redocusarus is just an integration for docusaurus with redoc. It doesn't add any features of it's own when it comes to rendering

I see there is already a related issue there - https://github.com/Redocly/redoc/issues/492
